docs/example-external.mdx

```diff
@@ -8,6 +8,8 @@ sidebar_label: External Examples
 
 - You can find more React Testing Library examples at
   [react-testing-examples.com](https://react-testing-examples.com/jest-rtl/).
+  
+  If the link does not work, try the netlify one: [react-testing-examples.netlify.app](https://react-testing-examples.netlify.app/).
```

Comment:
Hey @tarjei sorry for the late response, I forgot about it.

I asked the author and this can point directly to https://react-testing-examples.netlify.app/ (https://github.com/skidding/react-testing-examples/issues/56)

Reply:
:+1:  I updated the PR